### ISSUE TITLE
Automatically (or semi-automatically) map TMPDIR into VM

### ISSUE THREAD

**Issue by rix0rrr** [runfinch/finch#297]
**What is the problem you're trying to solve?.**

CDK will sometimes put directories with Dockerfiles into a user's temporary directory, and ask finch to build them. Right now, since the user's temporary directory isn't mounted into the finch VM, they get a confusing error like:

```
time="2023-03-16T09:43:07Z" level=fatal msg="lstat /private/var/folders/zx/d5wln9n10sn0tcj1v9798f1c0000gr/T/jsii-kernel-9VYgrO/node_modules/@aws-cdk/aws-lambda-python-alpha/lib/Containerfile: no such file or directory"
```

It's confusing on many levels:

- There is indeed no `Containerfile`, but there is a `Dockerfile`
- It's not obvious to a user that this path is inside the finch VM, instead of their own file system

**Describe the feature you'd like**

Right now we're telling users to put this into their `finch.yaml`:

```yaml
additional_directories:
  - path: /var/folders
  - path: /private/var/folders
```

It would be nice if finch could do this automatically.

If not, I wonder if there are performance implications of mapping everything under `/var/folders`. If so, could finch maybe support a symbolic name for the temporary directory instead?

```yaml
additional_directories:
  - path: TMPDIR
```

And either map that automatically always, or give the user the ability to add the above.

**Additional context**

Our issue: https://github.com/aws/aws-cdk/issues/24458

**Comment by rix0rrr** [comment]
Slight complication in the fact that:

```
$TMPDIR contains /var/folders/zx/d5wln9n10sn0tcj1v9798f1c0000gr/T/
```

But actually

```
/var -> /private/var
```

And the actual directory (that should probably be mapped) is:

```
/private/var/folders/zx/d5wln9n10sn0tcj1v9798f1c0000gr/T/
```

But now, unless the VM also has the `/var -> /private/var` symlink, it matters whether we use the unresolved `$TMPDIR` or the resolved `$TMPDIR`, because they will contain different path names and only one of them will be mapped...

**Comment by vsiravar** [comment]
Released in[ v0.8.0](https://github.com/runfinch/finch/releases/tag/v0.8.0)
